README.md

```diff
@@ -172,10 +172,48 @@ Verify JWT claims by setting JWT_CLAIM_VALIDATORS:
+| ClientJoinRateLimitReached         | The rate of joins per second from your clients as reached the channel limits                                                        | Contact support to increase your rate limits                                                       |
+| UnableToConnectToTenantDatabase    | Realtime was not able to connect to the tenant's database                                                                           | Contact support for further instructions                                                           |
+| RealtimeNodeDisconnected           | Realtime is a distributed application and this means that one the system is unable to communicate with one of the distributed nodes | Contact support for further instructions                                                           |
+| MigrationsFailedToRun              | Error when running the migrations against the Tenant database that are required by Realtime                                         | Contact support for further instructions                                                           |
+| ErrorStartingPostgresCDCStream     | Error when starting the Postgres CDC stream which is used for Postgres Changes                                                      | Contact support for further instructions                                                           |
+| UnknownDataProcessed               | An unknown data type was processed by the Realtime system                                                                           | Contact support for further instructions                                                           |
+| ErrorStartingPostgresCDC           | Error when starting the Postgres CDC extension which is used for Postgres Changes                                                   | Contact support for further instructions                                                           |
+| ReplicationSlotBeingUsed           | The replication slot is being used by another transaction                                                                           | Contact support for further instructions                                                           |
+| PoolingReplicationPreparationError | Error when preparing the replication slot                                                                                           | Contact support for further instructions                                                           |
+| PoolingReplicationError            | Error when pooling the replication slot                                                                                             | Contact support for further instructions                                                           |
```

Comment:
You mean Polling? Not Pooling?

Reply:
Also why not snake_case?

Reply:
I guess it's like a `defexception` where the exception is the module name?

Reply:
We're complying with the other error codes in Supabase ,that way we keep a single format in use